src/core.js

```diff
@@ -20,11 +20,11 @@ define( [
 	"./var/isWindow",
 	"./core/DOMEval",
 	"./core/toType"
-], function( arr, document, getProto, slice, concat, push, indexOf,
+], function(arr, document, getProto, slice, concat, push, indexOf,
```

Comment:
format: style mismatch:
```suggestion
], function( arr, document, getProto, slice, concat, push, indexOf,
```

`55a8147f` ␣ should be inserted at column 13.